app/src/main/java/com/kickstarter/viewmodels/PaymentMethodsViewHolderViewModel.kt

```diff
@@ -0,0 +1,101 @@
+package com.kickstarter.viewmodels
+
+import UserPaymentsQuery
+import com.kickstarter.R
+import com.kickstarter.libs.ActivityViewModel
+import com.kickstarter.libs.Environment
+import com.kickstarter.ui.viewholders.PaymentMethodsViewHolder
+import rx.Observable
+import rx.subjects.BehaviorSubject
+import rx.subjects.PublishSubject
+import type.CreditCardPaymentType
+import type.CreditCardTypes
+import java.text.SimpleDateFormat
+import java.util.*
+
+interface PaymentMethodsViewHolderViewModel {
+
+    interface Inputs {
+        /** Adding a card from the list of cards in the activity. */
+        fun card(creditCard: UserPaymentsQuery.Node)
+    }
+
+    interface Outputs {
+        /** Emits the expiration date for a credit card. */
+        fun expirationDate(): Observable<String>
+
+        /** Emits the credit card id. */
+        fun id(): Observable<String>
+
+        /** Emits the last four digits of the credit card. */
+        fun lastFour(): Observable<String>
+
+        /** Emits the payment type ex) Credit_Card or Bank_Account. */
+        fun paymentType(): Observable<CreditCardPaymentType>
+
+        /** Emits the card issuer ex) Visa or Mastercard. */
+        fun type(): Observable<Int>
+    }
+
+    class ViewModel(environment: Environment) : ActivityViewModel<PaymentMethodsViewHolder>(environment), Inputs, Outputs {
+
+        private val card = PublishSubject.create<UserPaymentsQuery.Node>()
+
+        private val expirationDate = BehaviorSubject.create<String>()
+        private val id = BehaviorSubject.create<String>()
+        private val lastFour = BehaviorSubject.create<String>()
+        private val paymentType = BehaviorSubject.create<CreditCardPaymentType>()
+        private val type = BehaviorSubject.create<Int>()
+
+        val inputs: Inputs = this
+        val outputs: Outputs = this
+
+        init {
+            this.card.map { expiration -> expiration.expirationDate() }
+                    .map {
+                        val sdf = SimpleDateFormat("MM/yyyy", Locale.getDefault())
```

Comment:
Let's make this a private variable so we don't recreate it every time `card` emits